docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -1,216 +1,339 @@
 ---
 title: Development workflow for Docker apps
-description: .NET Microservices Architecture for Containerized .NET Applications | Development workflow for Docker apps
+description: .NET Microservices Architecture for Containerized .NET Applications | Understand the details of the workflow for developing Docker-based applications. Begin step by step and get into some details to optimize Dockerfiles and end with the simplified workflow available when using Visual Studio
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 09/05/2018
+ms.date: 09/27/2018
 ---
 # Development workflow for Docker apps
 
-The application development life cycle starts at each developer’s machine, where the developer codes the application using their preferred language and tests it locally. No matter which language, framework, and platform the developer chooses, with this workflow, the developer is always developing and testing Docker containers, but doing so locally.
+The application development life cycle starts at each developer's machine, where the developer codes the application using their preferred language and tests it locally. No matter which language, framework, and platform the developer chooses, with this workflow, the developer is always developing and testing Docker containers, but doing so locally.
 
 Each container (an instance of a Docker image) includes the following components:
 
-- An operating system selection (for example, a Linux distribution, Windows Nano Server, or Windows Server Core).
+-   An operating system selection (for example, a Linux distribution, Windows Nano Server, or Windows Server Core).
 
-- Files added by the developer (application binaries, etc.).
+-   Files added by the developer (application binaries, etc.).
 
-- Configuration information (environment settings and dependencies).
+-   Configuration information (environment settings and dependencies).
 
 ## Workflow for developing Docker container-based applications
 
-This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not taking into account the broader DevOps workflow and just focuses on the development work done on the developer’s computer. The initial steps to set up the environment are not included, since those are done only once.
+This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not considering the broader DevOps workflow (that can include up to production deployment) and just focuses on the development work done on the developer's computer. The initial steps to set up the environment are not included, since those are done only once.
 
 An application is composed of your own services plus additional libraries (dependencies). The following are the basic steps you usually take when building a Docker application, as illustrated in Figure 5-1.
```

Comment:
not related to your PR but good for accessibility: 
The following are the basic steps you -> The following illustration shows the basic steps that you
Or "Figure 5-1 shows..."

and then remove , as illustrated in Figure 5-1